.cirrus.yml

```diff
@@ -2,14 +2,10 @@ gcp_credentials: ENCRYPTED[987a78af29b91ce8489594c9ab3fec21845bbe5ba68294b8f6def
 
 # LINUX
 task:
-  gke_container:
-    dockerfile: "ci/docker/build/Dockerfile"
-    builder_image_name: docker-builder # gce vm image
```

Comment:
I believe if we take this line out, we will have to start manually building and pushing the docker image again (which also I think won't work as is because we got rid of the GCP storage bucket we were pushing to). This task used the cached docker image, but the next time we change the dockerfile i suspect it will fail to build a new one. We should chat about our options on Monday.